plugins/modules/redis_info.py

```diff
@@ -2,49 +2,69 @@
 # -*- coding: utf-8 -*-
 
 # Copyright (c) 2020, Pavlo Bashynskyi (@levonet) <levonet@gmail.com>
-# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# Copyright (c) 2023, Xcelirate (@vicmunoz) <victor1 at teamxcl dot com>
+# GNU General Public License v3.0+
+# (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
 
-DOCUMENTATION = r'''
+DOCUMENTATION = '''
```

Comment:
Please keep the `r` for raw strings (same for the other two you removed).
```suggestion
DOCUMENTATION = r'''
```